### DISCUSSION TITLE
[Bug report] Publii not able to change to new loacation of sites

### DISCUSSION THREAD

**Discussion by tigexplorer** [GetPublii/Publii#1446] [Bug Report]
### Operating system

ubuntu 22.4

### Publii version

0.45.2

### Issue type

Application

### Bug description

The local working folder of my Publii website has changed.
I'm not able to edit it with Publii anymore.

If I start Publii, it is reporting, that the folder is not available and I have to make it manually. After this, several errors are thrown.

I mean, in that case, Publii must be able to provide a directory chooser to change the location to the new one.

### Relevant log output

```shell
In `.config/Publii/logs/` are no logs for this.
```

**Comment by s3nnet** [comment]
In the Publii/config folder edit the file app-config.json and change the line "sites folder" according to your new path
